docs/user_inferface/how-to-use-ianvs-command-line.md

```diff
@@ -0,0 +1,18 @@
+# How to use Ianvs command line
+
+## Ianvs
```

Comment:
It is suggested to list diffrent commands independently as sections

Reply:
fix it